docs/source/features.rst

```diff
@@ -3,15 +3,44 @@ Features
 
 Dynamic Source Tree Detection
 =============================
-coming soon
+Fab analyses your source code to determine the Fortran compile order,
+which means you don't have to manually specify and maintain the compile order,
+which can become problematic in a large project.
+
 
 Incremental Build
 =================
-coming soon
+Fab "watermarks" each artefact with a checksum of its inputs.
+Subsequent builds avoid reprocessing by searching for watermarks in the prebuild folder.
```

Comment:
Every build project puts its shareable, prebuilt artefacts into a folder called `_prebuild`, in the `build_output` folder. That's what other people can copy, and that's where we copy prebuilds from other people. It's just our incremental build folder, so perhaps it should be renamed to that?

It's not a two-stage process because everything which can be reused is in that folder. The parent `build_output` folder also contains some non-shareable things but we don't search for anything in two places because everything has one place to live. Happy to be steered on this, of course.